@xen-orchestra/lite/src/components/pool/system/PoolStorageConfiguration.vue

```diff
@@ -0,0 +1,91 @@
+            {{ defaultSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('suspend-storage-repository')">
+        <template #value>
+          <UiLink v-if="suspendSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ suspendSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('crash-dump-storage-repository')">
+        <template #value>
+          <UiLink v-if="crashDumpSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ crashDumpSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('heartbeat-storage-repository')">
+        <template #value>
+          <ul v-if="haSrs !== undefined && haSrs.length > 0">
+            <li v-for="haSr in haSrs" :key="haSr.uuid">
+              <UiLink v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+                {{ haSr.name_label }}
+              </UiLink>
+            </li>
+          </ul>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+    </template>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XenApiPool, XenApiSr } from '@/libs/xen-api/xen-api.types'
+import { useSrStore } from '@/stores/xen-api/sr.store'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import VtsLoadingHero from '@core/components/state-hero/VtsLoadingHero.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { faDatabase } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+const { pool } = defineProps<{
+  pool: XenApiPool
+}>()
+
+const { getByOpaqueRef, isReady } = useSrStore().subscribe()
+
+const defaultSr = computed(() => (pool.default_SR ? getByOpaqueRef(pool.default_SR) : undefined))
+const suspendSr = computed(() => (pool.suspendSr ? getByOpaqueRef(pool.suspendSr) : undefined))
+const crashDumpSr = computed(() => (pool.crash_dump_SR ? getByOpaqueRef(pool.crash_dump_SR) : undefined))
+
+const haSrs = computed(() => {
+  const haSrs = pool.ha_statefiles.filter(vdi => vdi !== undefined)
+
+  if (haSrs === undefined || haSrs?.length === 0) {
+    return
+  }
+
+  return haSrs.reduce((acc, srId) => {
+    const sr = getByOpaqueRef(srId)
+
+    return sr ? [...acc, sr] : acc
+  }, [] as XenApiSr[])
+})
```

Comment:
If I understood the code in `xapi-object-to-xo.mjs` correctly, in `ha_statefiles`, the values are VDIs refs (which you correctly understood as the variable naming in the filter suggests).
But then, we need to get the VDIs objects from these refs to finally retrieve the SR related to each VDI.
```suggestion
const { getByOpaqueRef: getVdiByOpaqueRef } = useVdiStore().subscribe()
// ...
const haSrs = computed(() => {
  const haSrs = pool.ha_statefiles.filter(vdi => vdi !== undefined)

  if (haSrs.length === 0) {
    return
  }

  return haSrs.reduce((acc, vdiRef) => {
    const vdi = getVdiByOpaqueRef(vdiRef)
    if (vdi === undefined) {
      return acc
    }

    const sr = getByOpaqueRef(vdi.SR)

    return sr ? [...acc, sr] : acc
  }, [] as XenApiSr[])
})
```